data/mainnet/tokens.js

```diff
@@ -83,6 +83,37 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'DON', 'WETH'],
     disableAutoSwap: true,
   },
+  rKP3R: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    logoUrl: './icons/rKP3R.png',
+    tokenAddress: addresses.rKP3R,
+    decimals: '18',
+    vaultAddress: null,
+    priceFunction: {
+      type: GET_PRICE_TYPES.COINGECKO_CONTRACT,
+      params: [addresses.KP3R],
+    },
+  },
+  crvIbEUR: {
+    isNew: true,
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: VAULT_CATEGORIES_IDS.STABLECOINS,
+    displayName: 'CRV:ibEUR',
+    apyIconUrls: ['./icons/curve.png', './icons/cvx.png', './icons/rKP3R.png'],
+    apyTokenSymbols: ['CRV', 'CVX', 'rKP3R'],
+    logoUrl: './icons/curve-ibeur.png',
+    tokenAddress: addresses.V2.convex_ibEUR.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.V2.convex_ibEUR.NewVault,
+    priceFunction: { type: GET_PRICE_TYPES.COINGECKO_CONTRACT, params: [addresses.ibEUR] },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.CONVEX,
+        params: ['f-ibeur', convexProfitSharingFactor],
```

Comment:
Use `strat30PercentFactor` instead of `convexProfitSharingFactor`. See discord discussion for reason.

Reply:
Depends on the strategy. The total fee taken of the convex strategies was different for the Convex strategies after we introduced the 10% hodl structure, so these strategies had a custom sharing factor. We now fixed it, so new strategies will just have the normal 30% fee.